tests-scan

```diff
@@ -123,13 +126,16 @@ def tests_human(priority, name, number, revision, ref, context, base, original_b
         branches=base != original_base and ("   {%s...%s}" % (original_base, base)) or ""
     )
 
+
 def is_internal_context(context):
     for pattern in ["rhel", "edge", "vmware", "openstack"]:
         if pattern in context:
             return True
     return False
 
 # Prepare a test invocation command
```

Comment:
I would keep function description right next to the function itself and put those two newlines above the description.